conf/distro/nilrt.inc

```diff
@@ -70,6 +70,13 @@ SDK_VERSION := "${@'${DISTRO_VERSION}'}"
 LOCALCONF_VERSION = "1"
 LAYER_CONF_VERSION ?= "5"
 
+# The console specified here is used as default by start_getty in case a real console
+# isn't detected from /proc/consoles (happens when console is disabled from MAX).
+# If a console isn't detected, we don't want start_getty to exit as that would
+# cause sysvinit to respawn it.
+# Using tty2 here as it isn't used elsewhere.
+SERIAL_CONSOLES = "38400;tty2"
```

Comment:
What overwrites this when console out is enabled?

Reply:
When console out is enabled, `/proc/consoles` is no longer empty and `start_getty` parses it and uses console specified there instead of the one here. Does that answer your question?